src/main/java/com/duke/task/Task.java

```diff
@@ -0,0 +1,40 @@
+package com.duke.task;
+
+import java.time.LocalDate;
+
+/**
+ * Represents a task specified by the user.
+ */
+public class Task {
+    protected String description;
+    public boolean isDone;
+
+    public Task(String d) {
```

Comment:
Is there a need for this particular constructor since Task is never explicitly used to construct an object?